test/jobs/create-initial-branch.js

```diff
@@ -162,6 +168,52 @@ test('create-initial-branch', async t => {
     t.end()
   })
 
+  t.test('badge already added for private repo', async t => {
```

Comment:
❓ why did you add this test?

Reply:
Same as https://github.com/greenkeeperio/greenkeeper/pull/602#discussion_r164702375